<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HabitTracker</class>
 <widget class="QWidget" name="HabitTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control de Hábitos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_habit">
     <property name="text">
      <string>Selecciona el hábito:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_habit">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radio_exercise">
        <property name="text">
         <string>Ejercicio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_reading">
        <property name="text">
         <string>Lectura</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_meditation">
        <property name="text">
         <string>Meditar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_frequency">
     <property name="text">
      <string>Selecciona la frecuencia:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="combo_frequency">
     <item>
      <property name="text">
       <string>Diariamente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Semanalmente</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mensualmente</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_count">
     <property name="text">
      <string>Ingresa el número de veces:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spin_count">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_submit">
     <property name="text">
      <string>Enviar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

